mix.exs

```diff
@@ -23,8 +23,8 @@ defmodule PhoenixPubsubRedis.Mixfile do
 
   defp deps do
     [{:phoenix_pubsub, "~> 1.0"},
-     {:redix, "~> 0.6.1"},
-     {:redix_pubsub, "~> 0.4.1"},
+     {:redix, "~> 0.8"},
```

Comment:
The deps should probably be locked to `"~> 0.8.1"` and `"~> 0.5.0"` as breaking changes could be introduced in a `0.9.0` or `0.6.0` future release in `redix` or `redis_pubsub`?

Reply:
Quite right. Fixed.